browse/static/js/core-recommender.js

```diff
@@ -0,0 +1,123 @@
+(function (window, userInput, options) {
+  if (userInput == null) userInput = {};
+  if (options == null) options = {};
+
+  // Options processing
+
+  var document = window.document;
+  var baseUrl = 'https://core.ac.uk';
+  var cookieName = options.cookieName || "arxiv_core_recommender";
+  var scriptId = options.scriptId || "recommender-embed";
+  var apiKey = options.apiKey || "24c597";
+
+
+  // Generic cookie manipulations
+
+  function setCookie(name, value, days) {
+    if (!name && !value) return false;
+  
+    if (days) {
+      var date = new Date();
+      date.setTime(date.getTime()+(days * 24 * 60 * 60 * 1000));
+      var expires = "; expires=" + date.toGMTString();
+    } else {
+      var expires = "";
+    }
+
+    document.cookie = name + "=" + value + expires +"; path=/";
+    return true;
+  }
+
+  function getCookie(name) {
+    var nameEq = name + "=";
+    var begin = document.cookie.indexOf(nameEq);
+    if (begin < 0) return null;
+
+    var end = document.cookie.indexOf(";", begin);
+    if (end < 0) end = document.cookie.length;
+
+    var value = decodeURIComponent(dc.substring(begin + nameEq.length, end));
+    return value;
+  }
+
+
+  // Generic DOM manipulations
+
+  function appendStylesheet(url, id) {
+    var linkElement = document.createElement('link');
+    linkElement.setAttribute('rel', 'stylesheet');
+    linkElement.setAttribute('type', 'text/css');
+    linkElement.setAttribute('href', url);
+    if (id != null) linkElement.id = id;
+    
+    document.head.appendChild(linkElement);
+    return linkElement;
+  }
+
+  function appendScript(url, id) {
+    var scriptElement = document.createElement('script');
+    scriptElement.type = "text/javascript";
+    scriptElement.src = url;
+    if (id != null) scriptElement.id = id;
+    
+    document.body.appendChild(scriptElement);
+    return scriptElement;
+  }
+
+
+  // CORE Recommender
+
+  function getRecommenderStatus() {
+    const cookieValue = getCookie(cookieName);
+    return cookieValue === "enabled";
```

Comment:
This assumes that if the cookie is not set (`getCookie` returns `null`), Recommender is not disabled. This means that the Recommender is disabled by default for all users. @mcancellieri should it be so?